contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-rule.y

```diff
@@ -268,6 +268,26 @@ tsql_CreateRoleStmt:
 				}
 			;
 
+tsql_CreatedbStmt:
+			CREATE DATABASE name opt_with createdb_opt_list
+				{
+					CreatedbStmt  *n = makeNode(CreatedbStmt);
+					n->dbname = $3;
+
+					/* If there are specified options, this is PSQL syntax */
```

Comment:
In case of TSQL CREATE DB, We can get non-null options list as well. We should remove this comment.

Reply:
done